lib/transport/data_validation/aggregates/project.ex

```diff
@@ -48,6 +57,11 @@ defmodule Transport.DataValidation.Aggregates.Project do
     GenServer.call(pid, {:find_feed_source, query})
   end
 
+  def execute(%FindFeedVersion{} = query) do
```

Comment:
How do we want to actually order actions?

It can be for example:

- Project -> FeedVersion -> FeedSource
- Query -> Command
- List -> Element

Reply:
I went for that. But `feedSource > feedVersion` as it is closer to the usual sequence